src/services/photos/database/photosRealmDB.ts

```diff
@@ -0,0 +1,170 @@
+import { BaseLogger } from '@internxt-mobile/services/common';
+import { fs } from '@internxt-mobile/services/FileSystemService';
+import { Photo, PhotoStatus } from '@internxt/sdk/dist/photos';
+import { Asset } from 'expo-media-library';
+import Realm, { ObjectSchema } from 'realm';
+
+// Store an alreay synced Photo
+const PhotoSchema: ObjectSchema = {
+  primaryKey: 'photoId',
+  properties: {
+    name: { type: 'string', indexed: true },
+    photoId: { type: 'string', indexed: true },
+    takenAt: 'date',
+    hash: 'string',
+    photoJSON: 'string',
+    status: 'string',
+  },
+  name: 'Photo',
+};
+
+type PhotoDoc = {
+  name: string;
+  photoId: string;
+  takenAt: Date;
+  hash: string;
+  status: PhotoStatus;
+  photoJSON: string;
+};
+
+// Store a DevicePhoto
+const DevicePhotoSchema: ObjectSchema = {
+  primaryKey: 'name',
+  properties: {
+    name: { type: 'string', indexed: true },
+    takenAt: 'date',
+    devicePhotoJSON: 'string',
+  },
+  name: 'DevicePhoto',
+};
+
+type DevicePhotoDoc = {
+  name: string;
+  takenAt: Date;
+  devicePhotoJSON: string;
+};
+export class PhotosRealmDB {
+  private _realm: Realm | null = null;
+  private logger: BaseLogger = new BaseLogger({
+    tag: 'PHOTOS_REALM_DB',
+    enabled: false,
+  });
+
+  async init() {
+    this._realm = await Realm.open({
```

Comment:
private is enough to make it private, using the `_something` convention is just for pure-JS code which wants to make it private as until [ES2022](https://betterprogramming.pub/es2022-a-complete-guide-on-private-static-class-fields-methods-and-static-initialization-blocks-f12943a86b2d) you were not able to make something private in a class